tests/main/security-internal-apparmor/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Check internal vendored apparmor in snapd snap
```

Comment:
I was thinking something like that but then I wondered if it was a bit of a misuse of the existing snapd-snap test to have it do a bunch of apparmor related bits - however other than duplicating it into this test I couldn't see a way to have one test use a snapd snap built from this PR.

But the idea of being able to test a snapd snap on various releases via lxd would be awesome - so thanks for working on that - then I can drop this one from here and put those bits in that test as you suggest.

Reply:
The PR is here: https://github.com/snapcore/snapd/pull/10099

Reply:
Well it took a while to land that PR, but now we have a PR which builds the snapd snap on various ubuntu-* systems at least (there are issues building with snapcraft and LXD on other distros), so you could add some of your desired tests from here to that tests/main/snapd-snap/task.yaml too

Reply:
Yep I added something already in https://github.com/snapcore/snapd/pull/9939/commits/8c77c3242baa302b396e653f121bc60633b027f4

Reply:
I think this particular spread test is somewhat redundant, because even if after this PR is merged, the test will not exercise the PR, it will simply be exercising the snapd snap on stable, which may be a useful/valid test but I think in this case the test should be a nightly test instead in the nightly suite under `tests/nightly`.

Reply:
I moved this test to nightly in https://github.com/snapcore/snapd/pull/9939/commits/e3f4b30f6f4e6dfe17c627e6290a86d3a0a969ac but have left it as manual for now since until this is actually merged to master it will not pass - once this gets merged then we can remove the manual declaration.